### ISSUE TITLE
local - if install broke after k3d create phase, the next execution will fail reclaiming k3d cluster already exist

### ISSUE THREAD

**Issue by pagottoo** [kubefirst/kubefirst#768]
```
LOG: 2022/11/18 16:30:51.429088 /Users/pagottoo/kubeshop/github/kubefirst/pkg/helpers.go:99: it is a yaml file, processing: /Users/pagottoo/.k1/gitops/terraform/vault/variables.tf
LOG: 2022/11/18 16:30:51.429178 /Users/pagottoo/kubeshop/github/kubefirst/internal/repo/kubefirstTemplate.go:73: detokenization of /Users/pagottoo/.k1/gitops complete
LOG: 2022/11/18 16:30:51.451477 /Users/pagottoo/kubeshop/github/kubefirst/internal/repo/kubefirstTemplate.go:85: git remote add github https://github.com/pagottoo/gitops
LOG: 2022/11/18 16:30:51.452061 /Users/pagottoo/kubeshop/github/kubefirst/internal/repo/kubefirstTemplate.go:117: committing detokenized gitops content
LOG: 2022/11/18 16:30:55.138233 /Users/pagottoo/kubeshop/github/kubefirst/cmd/local/prerun.go:156: clone and detokenization of gitops-template repository complete
LOG: 2022/11/18 16:30:55.138274 /Users/pagottoo/kubeshop/github/kubefirst/cmd/local/prerun.go:159: sending init completed metric
LOG: 2022/11/18 16:30:55.138280 /Users/pagottoo/kubeshop/github/kubefirst/pkg/helpers.go:593: init is done!

LOG: 2022/11/18 16:31:00.768676 /Users/pagottoo/kubeshop/github/kubefirst/cmd/local/local.go:101: Installing Github version of Kubefirst
LOG: 2022/11/18 16:31:00.768913 /Users/pagottoo/kubeshop/github/kubefirst/internal/k3d/create.go:16: Create K3d cluster for local install
LOG: 2022/11/18 16:31:01.059691 /Users/pagottoo/kubeshop/github/kubefirst/pkg/shell.go:22: Error executing command: exit status 1
LOG: 2022/11/18 16:31:01.060279 /Users/pagottoo/kubeshop/github/kubefirst/pkg/shell.go:24: Command Execution: /Users/pagottoo/.k1/tools/k3d
LOG: 2022/11/18 16:31:01.060304 /Users/pagottoo/kubeshop/github/kubefirst/pkg/shell.go:25: OUT:
LOG: 2022/11/18 16:31:01.060322 /Users/pagottoo/kubeshop/github/kubefirst/pkg/shell.go:26: ERR: FATA[0000] Failed to create cluster 'kubefirst' because a cluster with that name already exists

LOG: 2022/11/18 16:31:01.060335 /Users/pagottoo/kubeshop/github/kubefirst/internal/k3d/create.go:31: error creating k3d cluster
```

If the installation will continue where the last attempt stopped, is expected to check if the k3d cluster exists.
PS: destroy/clean cmd didn't destroy k3d cluster

**Comment by timoguin** [comment]
This has bit me multiple times during initial tests of running the local cluster. The only way I've managed to fix it is by using `k3d` to delete the cluster manually:

```
$ k3d cluster delete kubefirst
```

I've had to do this 4-5 times over two days trying to get it working. Seems like it should just resume operations when there is an existing cluster, or at least have an option for it. Do we just need to check and continue instead of fail?
